main.py

```diff
@@ -86,6 +86,11 @@ class MechaClient(Client):
             log.debug("received message from myself ignoring!.")
             return None
 
+        if not str(message).startswith(Commands.prefix):
+            # prevent bot from processing commands without the set prefix
+            log.debug(f"caught a command without prefix. Ignoring.")
```

Comment:
How do we know it is a command, specifically if it does not have the prefix?
May i suggest rewording it to something more along the lines of 
```py 
log.debug(f"ignoring message {message} as it does not start with my prefix")
```

Reply:
Updated.